### PR TITLE
DataGrid - Grouped rows and cells must have correct aria attributes

### PR DESCRIPTION
**Pull request by Raushen**
Removed not allowed aria attributes, added aria-roledescriprion attributes instead

### CODE REVIEW

File: js/localization/messages/ar.json [hunk 1]
```diff
@@ -133,6 +133,9 @@
         "dxDataGrid-sortingAscendingText": "فرز تصاعدي",
         "dxDataGrid-sortingDescendingText": "ترتيب تنازلي",
         "dxDataGrid-sortingClearText": "مسح الفرز",
+        "dxDataGrid-ariaNotSortedColumn": "Not sorted column",
+        "dxDataGrid-ariaSortedAscendingColumn": "Sorted column in ascending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/ar.json]:
```suggestion
        "dxDataGrid-ariaSortedAscendingColumn": "Column sorted in ascending order",
```

File: js/localization/messages/ar.json [hunk 2]
```diff
@@ -133,6 +133,9 @@
...
+        "dxDataGrid-ariaSortedDescendingColumn": "Sorted column in descending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/ar.json]:
```suggestion
        "dxDataGrid-ariaSortedDescendingColumn": "Column sorted in descending order",
```

File: js/localization/messages/ca.json [hunk 3]
```diff
@@ -133,6 +133,9 @@
         "dxDataGrid-sortingAscendingText": "Ordena Ascending",
         "dxDataGrid-sortingDescendingText": "Ordena descendent",
         "dxDataGrid-sortingClearText": "Ordenació clara",
+        "dxDataGrid-ariaNotSortedColumn": "Not sorted column",
+        "dxDataGrid-ariaSortedAscendingColumn": "Sorted column in ascending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/ca.json]:
```suggestion
        "dxDataGrid-ariaSortedAscendingColumn": "Column sorted in ascending order",
```

File: js/localization/messages/ca.json [hunk 4]
```diff
@@ -133,6 +133,9 @@
...
+        "dxDataGrid-ariaSortedDescendingColumn": "Sorted column in descending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/ca.json]:
```suggestion
        "dxDataGrid-ariaSortedDescendingColumn": "Column sorted in descending order",
```

File: js/localization/messages/cs.json [hunk 5]
```diff
@@ -133,6 +133,9 @@
         "dxDataGrid-sortingAscendingText": "Srovnat vzestupně",
         "dxDataGrid-sortingDescendingText": "Srovnat sestupně",
         "dxDataGrid-sortingClearText": "Zrušit rovnání",
+        "dxDataGrid-ariaNotSortedColumn": "Not sorted column",
+        "dxDataGrid-ariaSortedAscendingColumn": "Sorted column in ascending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/cs.json]:
```suggestion
        "dxDataGrid-ariaSortedAscendingColumn": "Column sorted in ascending order",
```

File: js/localization/messages/cs.json [hunk 6]
```diff
@@ -133,6 +133,9 @@
...
+        "dxDataGrid-ariaSortedDescendingColumn": "Sorted column in descending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/cs.json]:
```suggestion
        "dxDataGrid-ariaSortedDescendingColumn": "Column sorted in descending order",
```

File: js/localization/messages/el.json [hunk 7]
```diff
@@ -133,6 +133,9 @@
         "dxDataGrid-sortingAscendingText": "Ταξινόμηση αύξουσα",
         "dxDataGrid-sortingDescendingText": "Ταξινόμηση φθήνουσα",
         "dxDataGrid-sortingClearText": "Εκκαθάριση ταξινόμισης",
+        "dxDataGrid-ariaNotSortedColumn": "Not sorted column",
+        "dxDataGrid-ariaSortedAscendingColumn": "Sorted column in ascending order",
```
Comment (dirk-pieterse) [inline js/localization/messages/el.json]: Same here and in other places

**Review by dirk-pieterse — DISMISSED** [review]

**Review by sergepilipchuk — APPROVED** [review]

File: js/__internal/grids/grid_core/sorting/m_sorting.ts [hunk 8]
```diff
@@ -120,6 +120,7 @@ const HeaderPanelSortingExtender = extend({}, sortingMixin, {
         alignment: that.option('rtlEnabled') ? 'right' : 'left',
         allowSorting: groupColumn.allowSorting,
         sortOrder: groupColumn.sortOrder === 'desc' ? 'desc' : 'asc',
+        isGrouped: true,
```
Comment (pomahtri) [inline js/__internal/grids/grid_core/sorting/m_sorting.ts:123]: question: is it a new property of column object, or did we forget to add this property here?

**Pull request closed without merging**